src/GraphQL.DataLoader.Tests/Types/OrderItemType.cs

```diff
@@ -17,7 +17,7 @@ public OrderItemType(IDataLoaderContextAccessor accessor, IProductsStore product
         Field<ProductType, Product>("Product")
             .ResolveAsync(ctx =>
             {
-                var loader = accessor.Context.GetOrAddBatchLoader<int, Product>("GetProductById",
+                var loader = accessor.Context!.GetOrAddBatchLoader<int, Product>("GetProductById",
```

Comment:
I believe these will be reverted in `develop` branch? https://github.com/graphql-dotnet/graphql-dotnet/pull/3792

Reply:
They are unnecessary in the develop branch, right.